app/services/tokens_repo.js

```diff
@@ -0,0 +1,26 @@
+const tokensRepo = [];
```

Comment:
i dont like this approach, is not pure, you have to maintain state here

Reply:
why dont you save them in the db?

Reply:
because:
* I much slower
* There are no multiple instances of my app running.
* The best approach would be to use a key value DB like Redis (But it's out of training's scope)